crates/notedeck_columns/src/post.rs

```diff
@@ -119,4 +137,45 @@ impl NewPost {
             .build()
             .expect("expected build to work")
     }
+
+    fn extract_hashtags(content: &str) -> Vec<String> {
+        let mut hashtags = Vec::new();
+        for word in content.split_whitespace() {
+            if word.starts_with('#') && word.len() > 1 {
+                let tag = word[1..].trim_end_matches(|c: char| !c.is_alphanumeric())
+                    .to_string();
+                if !tag.is_empty() {
+                    hashtags.push(tag);
+                }
+            }
+        }
+        hashtags
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_extract_hashtags() {
+        let test_cases = vec![
+            ("Hello #world", vec!["world"]),
+            ("Multiple #tags #in #one post", vec!["tags", "in", "one"]),
+            ("No hashtags here", vec![]),
+            ("#tag1 with #tag2!", vec!["tag1", "tag2"]),
+            ("Ignore # empty", vec![]),
+            ("Keep #alphanumeric123", vec!["alphanumeric123"]),
+        ];
```

Comment:
can you add tests for emojis

Reply:
Good catch on the emoji hashtag, had to fix that case